frontend/packages/kubevirt-plugin/src/components/vm-snapshots/types.ts

```diff
@@ -6,4 +7,6 @@ export type VMSnapshotRowCustomData = {
   vmLikeEntity: VMLikeEntityKind;
   columnClasses: string[];
   isDisabled: boolean;
+  restores: VMRestore[];
+  withProgress: (promise: Promise<any>) => void;
```

Comment:
why do we need withProgress in snapshots?

Reply:
why not? its a resource creation like any other modal?

Reply:
because this feature is used only by the wizard

Reply:
ah you are right, It is used by the detail pages as well